### PR TITLE
We should remove the address field from the pCDN wallet if the addres…

### PR DESCRIPTION
**Pull request by tsmartt**
…s is nil

### CODE REVIEW

File: app/jobs/cache/browser_channels/responses_for_prefix.rb [hunk 1]
```diff
@@ -63,7 +66,10 @@ def generate_brotli_encoded_channel_response(prefix:)
           gemini_wallet.wallet_state = get_gemini_wallet_state(gemini_connection: connection)
 
           if connection.country && allowed_regions[:gemini][:allow].include?(connection.country.upcase)
-            gemini_wallet.address = site_banner_lookup.channel.gemini_connection&.recipient_id
+            gemini_address = site_banner_lookup.channel.gemini_connection&.recipient_id
```
Comment (OAuthBringer) [inline app/jobs/cache/browser_channels/responses_for_prefix.rb:69]: @tsmartt Here as well.

**Comment by OAuthBringer** [comment]
@tsmartt Why would we include the channel in the pCDN at all if it has no address?

**Review by OAuthBringer — CHANGES_REQUESTED** [review]
I'm not sure where the change is, but we should revert to having `address` set to empty string and not dropped as it would likely break everything.

File: app/jobs/cache/browser_channels/responses_for_prefix.rb [hunk 2]
```diff
@@ -40,7 +40,10 @@ def generate_brotli_encoded_channel_response(prefix:)
           uphold_wallet.wallet_state = get_uphold_wallet_state(uphold_connection: connection)
 
           if connection.country && allowed_regions[:uphold][:allow].include?(connection.country.upcase)
-            uphold_wallet.address = site_banner_lookup.channel.uphold_connection&.address
+            uphold_address = site_banner_lookup.channel.uphold_connection&.address
```
[the remark this answers is not in the document]
Comment (tsmartt) [reply app/jobs/cache/browser_channels/responses_for_prefix.rb]: Resolved

File: app/jobs/cache/browser_channels/responses_for_prefix.rb (hunk 1 above)
Comment (tsmartt) [reply app/jobs/cache/browser_channels/responses_for_prefix.rb]: Resolved

**Comment by tsmartt** [comment]
> I'm not sure where the change is, but we should revert to having `address` set to empty string and not dropped as it would likely break everything.

They are now included, but with a blank string

**Review by OAuthBringer — APPROVED** [review]

**Pull request merged**